docs/settings/reporting-settings.asciidoc

```diff
@@ -203,26 +203,35 @@ The `file:` protocol is always denied, even if no network policy is configured.
 ==== CSV settings
 
 [[xpack-reporting-csv]] `xpack.reporting.csv.maxSizeBytes` {ess-icon}::
-The maximum {byte-units}[byte size] of a CSV file before being truncated. This setting exists to prevent large exports from causing performance and storage issues. Can be specified as number of bytes. Defaults to `10mb`.
+The maximum {byte-units}[byte size] of a CSV file before being truncated. This setting exists to prevent large exports from causing performance and storage issues. Can be specified as number of bytes. Defaults to `250mb`.
```

Comment:
Based on https://github.com/elastic/kibana/issues/159514, we haven't actually updated the default limit yet. I'd remove this for now.

Reply:
I'm at a point where I can fast-track this change, as well as making `xpack.reporting.csv.scroll.duration` to be a user-editable setting in ESS. I will update here, in case the change is made while this PR is still open.

Reply:
The default for maxSizeBytes will be increased in 8.10: https://github.com/elastic/kibana/pull/161318